### PR TITLE
Update Group Info at VPN Net Access level

### PR DESCRIPTION
**Pull request by sbarguil**

### CODE REVIEW

**Comment by sbarguil** [comment]
ESI at Vpn Net acces level as discussed in the meeting https://codimd.ietf.org/40Dw-yCkQoONcMsYp8e5XQ?both

``` 
                      |  +--rw access-diversity {vpn-common:placement-diversity}?
                       |  |  +--rw groups
                       |  |     +--rw group* [group-id]
                       |  |        +--rw group-id                       string
                       |  |        +--rw fate-sharing-group-size?       uint16
                       |  |        +--rw group-color?                   string
                       |  |        +--rw ethernet-segment-identifier?   yang:hex-string
```

File: yang/ietf-l2vpn-ntw.yang [hunk 1]
```diff
@@ -141,6 +141,22 @@ module ietf-l2vpn-ntw {
   /* Typedefs */
   /* Identities */
 
+  identity multi-homing-types {
+    description "Base identity for a EVPN capability";
+  }
+
+  identity SINGLE_ACTIVE {
```
Comment (boucadair) [inline yang/ietf-l2vpn-ntw.yang:148]:
```suggestion
  identity single-active {
```

File: yang/ietf-l2vpn-ntw.yang [hunk 2]
```diff
@@ -141,6 +141,22 @@ module ietf-l2vpn-ntw {
...
+    base multi-homing-types;
+    description
+      "Single-active multi-homing ethernet-segment";
+  }
+
+  identity ALL_ACTIVE {
```
Comment (boucadair) [inline yang/ietf-l2vpn-ntw.yang:154]:
```suggestion
  identity all-active {
```

File: yang/ietf-l2vpn-ntw.yang [hunk 3]
```diff
@@ -141,6 +141,22 @@ module ietf-l2vpn-ntw {
   /* Typedefs */
   /* Identities */
 
+  identity multi-homing-types {
```
Comment (boucadair) [inline yang/ietf-l2vpn-ntw.yang:144]:
```suggestion
  identity evpn-redundancy-mode {
```

File: yang/ietf-l2vpn-ntw.yang [hunk 4]
```diff
@@ -141,6 +141,22 @@ module ietf-l2vpn-ntw {
...
+    description "Base identity for a EVPN capability";
+  }
+
+  identity SINGLE_ACTIVE {
+    base multi-homing-types;
```
Comment (boucadair) [inline yang/ietf-l2vpn-ntw.yang:149]:
```suggestion
    base evpn-redundancy-mode;
```

File: yang/ietf-l2vpn-ntw.yang [hunk 5]
```diff
@@ -141,6 +141,22 @@ module ietf-l2vpn-ntw {
...
+    description
+      "Single-active multi-homing ethernet-segment";
+  }
+
+  identity ALL_ACTIVE {
+    base multi-homing-types;
```
Comment (boucadair) [inline yang/ietf-l2vpn-ntw.yang:155]:
```suggestion
    base evpn-redundancy-mode;
```

File: yang/ietf-l2vpn-ntw.yang [hunk 6]
```diff
@@ -141,6 +141,22 @@ module ietf-l2vpn-ntw {
...
   /* Identities */
 
+  identity multi-homing-types {
+    description "Base identity for a EVPN capability";
+  }
+
+  identity SINGLE_ACTIVE {
+    base multi-homing-types;
+    description
+      "Single-active multi-homing ethernet-segment";
```
Comment (boucadair) [inline yang/ietf-l2vpn-ntw.yang:151]:
```suggestion
      "Indicates Single-Active redundancy mode for 
       a given Ethernet Segment (ES).";
     reference
       "RFC 7432: BGP MPLS-Based Ethernet VPN, Section-14.1.1";
```

File: yang/ietf-l2vpn-ntw.yang [hunk 7]
```diff
@@ -141,6 +141,22 @@ module ietf-l2vpn-ntw {
...
+  }
+
+  identity ALL_ACTIVE {
+    base multi-homing-types;
+    description
+      "All-active multi-homing ethernet-segment";
```
Comment (boucadair) [inline yang/ietf-l2vpn-ntw.yang:157]:
```suggestion
      "Indicates All-Active redundancy mode for 
       a given Ethernet Segment (ES).";
     reference
       "RFC 7432: BGP MPLS-Based Ethernet VPN, Section-14.1.2";
```

File: yang/ietf-l2vpn-ntw.yang [hunk 8]
```diff
@@ -1171,8 +1144,32 @@ module ietf-l2vpn-ntw {
...
+                      leaf ethernet-segment-identifier {
+                        type yang:hex-string {
+                          length "29";
+                        }
+                        description
+                          "10-octet Ethernet Segment Identifier (ESI).";
+                      }
+                      leaf multi-homing {
+                        type identityref{
+                          base multi-homing-types;
```
Comment (boucadair) [inline yang/ietf-l2vpn-ntw.yang:1166]:
```suggestion
                          base evpn-redundancy-mode;
```

File: yang/ietf-l2vpn-ntw.yang [hunk 9]
```diff
@@ -1171,8 +1144,32 @@ module ietf-l2vpn-ntw {
...
+                        }
+                        description "Multihoming options for the evpn";
```
Comment (boucadair) [inline yang/ietf-l2vpn-ntw.yang:1168]:
```suggestion
                        description 
                          "Indicates the EVPN redundancy mode for
                           a multihomed CE.";
```

File: yang/ietf-l2vpn-ntw.yang [hunk 10]
```diff
@@ -1171,8 +1144,32 @@ module ietf-l2vpn-ntw {
...
+                          "Group color associated with a particular VPN.";
+                      }
+                      leaf ethernet-segment-identifier {
+                        type yang:hex-string {
+                          length "29";
+                        }
+                        description
+                          "10-octet Ethernet Segment Identifier (ESI).";
+                      }
+                      leaf multi-homing {
```
Comment (boucadair) [inline yang/ietf-l2vpn-ntw.yang:1164]:
```suggestion
                      leaf redundancy-mode {
```

**Review by boucadair — CHANGES_REQUESTED** [review]
Some proposals below

File: yang/ietf-l2vpn-ntw.yang [hunk 11]
```diff
@@ -141,6 +141,28 @@ module ietf-l2vpn-ntw {
   /* Typedefs */
   /* Identities */
 
+  identity evpn-redundancy-mode {
+    description "Base identity for a EVPN capability";
```
Comment (boucadair) [inline yang/ietf-l2vpn-ntw.yang:145]:
```suggestion
    description 
      "Base identity for a EVPN redundancy modes.";
```

File: yang/ietf-l2vpn-ntw.yang [hunk 12]
```diff
@@ -141,6 +141,29 @@ module ietf-l2vpn-ntw {
   /* Typedefs */
   /* Identities */
 
+  identity evpn-redundancy-mode {
+    description 
+      "Base identity for a EVPN redundancy modes.";
```
Comment (boucadair) [inline yang/ietf-l2vpn-ntw.yang:146]:
```suggestion
      "Base identity for EVPN redundancy modes.";
```

**Review by boucadair — APPROVED** [review]
Thanks, Samir.

**Pull request merged**